src/cpr_sdk/pipeline_general_models.py

```diff
@@ -31,11 +32,15 @@ class BackendDocument(BaseModel):
     date: Optional[str] = None  # Set on import by a validator
     source_url: Optional[str] = None
     download_url: Optional[str] = None
-
+    corpus_import_id: Optional[str] = ""
+    corpus_type_name: Optional[str] = ""
+    collection_title: Optional[str] = ""
+    collection_summary: Optional[str] = ""
     type: str
     source: str
     category: str
     geography: str
+    geographies: list[str] = []
```

Comment:
**thought:** Is there a difference between an empty list and an optional list (which may be empty)? 🤔 (This is just a thought to be ignored)

Reply:
Thinking about it, maybe this is the approach we want above too... It basically sets `[]` as a default value if no other value is provided. But without allowing the value to be `None`.

Reply:
The general data modelling approach I take is that `None` of something is different to the empty value (`""` or `[]`) of something, and thus prefer `Optional`s over potentially misleading empty values. For example, is the value missing or empty? With Only a list, we'll never know.

I don't have a strong opinion on it for this data modelling decision though, so happy with either approach.

Reply:
Taken this approach, thanks! 👍